src/cli/commands/app.rs

```diff
@@ -0,0 +1,17 @@
+use clap::Subcommand;
+
+#[derive(Subcommand)]
+pub(crate) enum Commands {
+    /// Create a new application
+    Add {
+        /// The name of the application
+        name: String,
+        /// The Bindle ID where releases will be uploaded
+        storage_id: String,
+    },
+    /// Remove an application
+    Remove {
+        /// The application ID
+        id: String,
```

Comment:
Creating an application works great — I can validate that in the UI:

```
❯ hippo app add testapp radu/testapp
Added App testapp (ID = '2f0c046f-ad29-4846-892b-b850e7d0eba7')
IMPORTANT: save this App ID for later - you will need it to update and/or delete the App (for now)
```

Deleting the app doesn't seem to work:

```
❯ hippo app remove 2f0c046f-ad29-4846-892b-b850e7d0eba7
Error: error in response: status code 404 Not Found
```

Reply:
Good catch. I think a 404 is the expected response, so maybe we need to capture that instead of casting via `?`

Reply:
Hmm... The server should be responding with a 204 no content according to https://github.com/deislabs/hippo/blob/bae252ef73038ec6e473d852c384aef70695e84d/src/Web/Api/AppController.cs#L41

Will look into this further

Reply:
This is happening in the UI as well, so I'm lead to believe this is a server-side issue. Once I determine the cause I'll try to fix it before merging this

Reply:
identified the issue in https://github.com/deislabs/hippo/pull/401 - misconfigured API routes